[types] Added types for JSON config of tabs

GermanBluefox opened the pull request:
## Added types for sendTo in a case of JSON config for tabs
I implemented new feature: https://github.com/ioBroker/ioBroker.admin/tree/master/packages/jsonConfig#json-tab-in-admin
For that the new flag must be added to io-package.json

## Formatting of objectsInRedisClient.ts
Only replaced `var && var.attr` with 'var?.attr`

**Tests**
- [ ] I have added tests to avoid a recursion of this bug
- [X] It is not possible to test for this bug. But I can compile it

**Documentation**
- [X] I have documented the new feature in schema

foxriver76 commented on schemas/io-package.json:
```diff
@@ -1130,6 +1130,17 @@
               "description": "If true, only one TAB for all instances will be shown.",
               "type": "boolean"
+            },
+            "sendTo": {
```
I don't get what this is doing by the description

foxriver76 replied:
Any why boolean or string this isn't the best design if this is newly introduced.

GermanBluefox replied:
How would you do it? This attribute describes if the message must be sent when the GUI initializing:
- true - send 'tab' message
- anyString - send 'anyString' message to backend, so it can create an answer for JSON Config

foxriver76 replied:
Still don’t get what it is doing, why isn’t it a json config flag but an io package flag? Why not set string to „tab“ to send „tab“?

GermanBluefox commented:
The idea to have sendTo flag in the json file was good, so I moved the flag there.

This PR is only code formatting and some changed comments. And removed some "required" attributes from `adminTab`
